changelog.md

```diff
@@ -183,3 +183,6 @@ with other backends. see #9125. Use `-d:nimLegacyJsRound` for previous behavior.
   - cell alignment is not supported, i.e. alignment annotations in a delimiter
     row (`:---`, `:--:`, `---:`) are ignored,
   - every table row must start with `|`, e.g. `| cell 1 | cell 2 |`.
+
+- `fusion` is now un-bundled from nim, `./koch fusion` will result in an error.
+  `nimble install fusion` should be used instead.
```

Comment:
I think 'koch fusion'  should be an alias for "nimble install fusion".

Reply:
fixed, but honoring a fixed hash and --latest